src/quo2/components/wallet/transaction_progress/view.cljs

```diff
@@ -0,0 +1,296 @@
+(ns quo2.components.wallet.transaction-progress.view
+  (:require [quo2.components.wallet.transaction-progress.style :as style]
+            [quo2.core :as quo2]
+            [quo2.components.selectors.selectors.view :as selectors]
+            [quo2.components.buttons.button.view :as button]
+            [quo2.components.wallet.progress-bar.view :as progress-box]
+            [quo2.components.markdown.text :as text]
+            [quo2.components.tags.status-tags :as status-tag]
+            [quo2.foundations.colors :as colors]
+            [status-im2.common.resources :as resources]
+            [quo2.theme :as theme]
+            [reagent.core :as reagent]
+            [react-native.core :as rn]))
+
+(defn get-colors
+  [color]
+  (let [current-theme (theme/get-theme)]
```

Comment:
Avoid getting the theme from `theme/get-theme`.

Please wrap the component with `quo.theme/with-theme` HOC to get the current theme from the theme provider.